src/assets/styles/variables.scss

```diff
@@ -20,6 +20,13 @@
   --paragraph-small: 0.8rem;
   --label: 0.64rem;
   --button: 1rem;
+  --line-height: normal;
+  --letter-spacing: 0;
+  --letter-spacing-eyebrow: 0.25rem;
```

Comment:
Not too sure about this one it doesn't feel generic enough to add here.

Reply:
the --letter-spacing-eyebrow: 0.25rem;

Reply:
My thinking is around clarity. The dev can know that everything in the Foundations is a variable in the codebase.
![Screenshot 2023-09-06 at 12 03 59 PM](https://github.com/CloudCannon/small-business-template-eleventy/assets/101225153/f5237d5a-ee3d-444a-a710-d5ef9c61cd6a)

But you are right its very specific and only used once (as far as I can see). I would be happy either way.

Reply:
I wonder if these should be here but made less specific? instead of `font-weight-button` and `font-weight-headings` it is just `font-weight-bold`, and instead of `letter-spacing-eyebrow` its `letter-spacing-wide`?

Reply:
The most recent commit changed the names to be more generic FYI